### PR TITLE
Chore: progress chart changes

### PR DESCRIPTION
**Pull request by gakshita**
## Release Notes

- **New Features**
  - Introduced `CycleSidebarHeader` and `CycleSidebarDetails` components for enhanced sidebar functionality in cycle management.
  - Added a new button for opening cycle overviews, improving navigation and user interaction.

- **Improvements**
  - Enhanced conditional rendering in cycle list actions for better user experience.

- **Bug Fixes**
  - Resolved issues with redundant exports in the analytics sidebar module.

- **Documentation**
  - Updated documentation for newly added components and features.

### CODE REVIEW

File: web/app/[workspaceSlug]/(projects)/projects/(detail)/[projectId]/cycles/(detail)/[cycleId]/page.tsx [hunk 1]
```diff
@@ -70,14 +70,14 @@ const CycleDetailPage = observer(() => {
...
-                  "flex h-full w-[24rem] flex-shrink-0 flex-col gap-3.5 overflow-y-auto border-l border-custom-border-100 bg-custom-sidebar-background-100 px-6 duration-300 vertical-scrollbar scrollbar-sm absolute right-0 z-[13]"
+                  "flex h-full w-[21.5rem] flex-shrink-0 flex-col gap-3.5 overflow-y-auto border-l border-custom-border-100 bg-custom-sidebar-background-100 px-4 duration-300 vertical-scrollbar scrollbar-sm absolute right-0 z-[13]"
                 )}
                 style={{
                   boxShadow:
                     "0px 1px 4px 0px rgba(0, 0, 0, 0.06), 0px 2px 4px 0px rgba(16, 24, 40, 0.06), 0px 1px 8px -1px rgba(16, 24, 40, 0.06)",
                 }}
               >
-                <CycleDetailsSidebar cycleId={cycleId.toString()} handleClose={toggleSidebar} />
+                <CycleDetailsSidebar handleClose={toggleSidebar} />
```
[the remark this answers is not in the document]
Comment (SatishGandham) [reply web/app/[workspaceSlug]/(projects)/projects/(detail)/[projectId]/cycles/(detail)/[cycleId]/page.tsx:80]: cc: @gakshita

File: web/core/components/cycles/active-cycle/productivity.tsx [hunk 2]
```diff
@@ -19,22 +19,22 @@ export type ActiveCycleProductivityProps = {
...
 export const ActiveCycleProductivity: FC<ActiveCycleProductivityProps> = observer((props) => {
   const { workspaceSlug, projectId, cycle } = props;
   // hooks
-  const { getPlotTypeByCycleId, setPlotType } = useCycle();
+  const { getEstimateTypeByCycleId, setEstimateType } = useCycle();
   const { currentActiveEstimateId, areEstimateEnabledByProjectId, estimateById } = useProjectEstimates();
 
   // derived values
-  const plotType: TCyclePlotType = (cycle && getPlotTypeByCycleId(cycle.id)) || "burndown";
+  const estimateType: TCycleEstimateType = (cycle && getEstimateTypeByCycleId(cycle.id)) || "issues";
```
Comment (SatishGandham) [inline web/core/components/cycles/active-cycle/productivity.tsx:33]: Will progressUnit be a better name?

File: web/core/components/cycles/analytics-sidebar/index.ts [hunk 3]
```diff
@@ -1,3 +1,6 @@
 export * from "./root";
 export * from "./issue-progress";
 export * from "./progress-stats";
+export * from "./root";
```
[the remark this answers is not in the document]
Comment (SatishGandham) [reply web/core/components/cycles/analytics-sidebar/index.ts:4]: cc: @gakshita

**Review by SatishGandham — CHANGES_REQUESTED** [review]
Please check the coderabbit suggestions and other comments

File: web/core/components/cycles/active-cycle/productivity.tsx (hunk 2 above)
Comment (gakshita) [reply web/core/components/cycles/active-cycle/productivity.tsx:33]: Its a judgement call, to me estimateType looks better

**Review by SatishGandham — APPROVED** [review]

**Pull request merged**